server/src/main/java/org/elasticsearch/http/AbstractHttpServerTransport.java

```diff
@@ -174,7 +174,10 @@ private TransportAddress bindAddress(final InetAddress hostAddress) {
             return true;
         });
         if (!success) {
-            throw new BindHttpException("Failed to bind to [" + port.getPortRangeString() + "]", lastException.get());
+            throw new BindHttpException(
```

Comment:
This is failing forbidden APIs checks, see the logs:


```
Forbidden method invocation: java.net.InetAddress#getHostAddress() [use NetworkAddress format/formatAddress to print IP or IP+ports]
  in org.elasticsearch.transport.TcpTransport (TcpTransport.java:381)
Forbidden method invocation: java.net.InetAddress#getHostAddress() [use NetworkAddress format/formatAddress to print IP or IP+ports]
  in org.elasticsearch.http.AbstractHttpServerTransport (AbstractHttpServerTransport.java:178)
Scanned 5715 class file(s) for forbidden API invocations (in 5.20s), 2 error(s).
```

Reply:
Hi @original-brownbear, thanks for reviewing this. I ran the tests but forgot to run the check task. Will fix this ASAP.

Reply:
@mariaral FYI, you can just run these simple static code checks via:

```
./gradlew precommit 
```

no need to run a full `check` for those

Reply:
Thanks @original-brownbear for the tip. I didn't know of the precommit task.

I'm thinking of extending `NetworkAddress.format()` to be able to handle PortsRange objects and use that instead of coping the same message in five places. I propose using the following format:
```
    IPv4 no port: 127.0.0.1
    IPv4 single port: 127.0.0.1:9300}
    IPv4 multiple ports: 127.0.0.1:[9300-9400]
    IPv6 multiple ports: [::1]:[9300-9400]
```
What do you think?

Reply:
I only count two spots where we print the `PortsRange` but fine by me to keep it consistent via a utility method :)

Reply:
I pushed a new commit. Please take a look.